rcon/extended_commands.py

```diff
@@ -998,7 +1004,7 @@ def set_maprotation(self, rotation):
         rotation = list(rotation)
         logger.info("Apply map rotation %s", rotation)
 
-        current = self.get_map_rotation()
+        current = super().get_map_rotation()
```

Comment:
as we rely on the current map rotation to rebuild the current one, I thought it would be wise to bypass any cached version we might have in case the server shuffled the map rotation or someone changed it with another rcon tool already.

Reply:
lol, `invalidates` invalidates the cache _before_ and _after_ executing, I didn't expect that :( Then I can move querying the current map rotation a bit down and it should be save as well.